### ISSUE TITLE
Update to 0.2.5 ... i got 2 warnings

### ISSUE THREAD

**Issue by roe1974** [nfarina/homebridge-sonos#67]
Hi

i just updated to 0.2.5 and got 2 warnings while installing:

npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated tough-cookie@2.2.2: ReDoS vulnerability parsing Set-Cookie https://nodesecurity.io/advisories/

ist this normal ?

Richard

**Comment by nfarina** [comment]
Yeah those are pretty minor warnings, nothing to worry about (in my opinion at least).
